src/components/utils/popover/popover.types.ts

```diff
@@ -54,9 +55,25 @@ export enum PopoverEvent {
   /**
    * When popover closes
    */
-  Close = 'close'
+  Close = 'close',
+
+  /**
+   * When it closes because item with 'activateOnClose' property set was clicked
+   */
+  CloseOnActivate = 'close-on-activate',
+
+  /**
+   * When nested popover should opens
+   */
+  OpenNestedPopover = 'open-nested-popover',
```

Comment:
Its better to name events in a past form

Reply:
updated events names